### DISCUSSION TITLE
Provide version for About dialog on Mac

### DISCUSSION THREAD

**Discussion by gitting** [ananthakumaran/paisa#71] [General]
Mac apps usually provide versions when asked `About <app>`. Currently paisa doesn't provide that as in:

<img width="372" alt="Screenshot 2023-10-07 at 12 11 27 AM" src="https://github.com/ananthakumaran/paisa/assets/3186756/bb2449c2-9f2d-4ebc-ae58-440975d17b11">

It would be good to have that.

**Comment by ananthakumaran** [comment]
<img width="270" alt="Screenshot 2023-10-07 at 5 39 42 PM" src="https://github.com/ananthakumaran/paisa/assets/149238/7338e98d-a983-4ed7-9c13-cc5a50b3a2d2">

Added to master, will be available from next release.

**Reply by avinashkanaujiya** [reply]
Same for linux, would be nice.

**Reply by ananthakumaran** [reply]
I have added a new about page https://demo.paisa.fyi/more/about.
